### PR TITLE
RELATED: RAIL-558 Fixed Drilling for adhoc measures in Table

### PR DESCRIPTION
**Pull request by vojty**
- [x] table
- [x] charts

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/287/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/897/ : FAILURE in 4m 38s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/165/ : FAILURE in 1m 34s (non-voting)

File: src/utils/drilldownEventing.js [hunk 1]
```diff
@@ -1,25 +1,48 @@
...
+            uri: get(measure, ['definition', 'measure', 'item', 'uri']),
+            identifier: get(measure, ['definition', 'measure', 'item', 'identifier'])
+        };
+    }
+    return null;
+}
+
+function isHeaderDrillable(drillableItems, header) {
+    return drillableItems.some(drillableItem =>
+        // Check for defined values because undefined === undefined OLOL
```
Comment (vaclavbohac) [inline src/utils/drilldownEventing.js]: Remove OLOL

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/291/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/899/ : SUCCESS in 1m 19s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/167/ : SUCCESS in 1m 33s (non-voting)

**Comment by vojty** [comment]
`1.0.0-vojty-drill-fix-2017-12-11T15-58-37-904Z`

**Comment by vaclavbohac** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/22/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/292/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/175/ : SUCCESS in 4m 06s

**Pull request merged**